tests/virtctl/ssh.go

```diff
@@ -74,12 +45,41 @@ var _ = Describe("[sig-compute][virtctl]SSH", func() {
 			"-t", "-o StrictHostKeyChecking=no",
 			"-t", "-o UserKnownHostsFile=/dev/null",
 			"--command", "true",
-			vmi.Name)
+			vmiName)
 		Expect(err).ToNot(HaveOccurred())
 		cmd.Env = append(cmd.Env,
```

Comment:
Is this required if we've already Expect(os.Setenv("SSH_AUTH_SOCK", "/dev/null")).To(Succeed()) in BeforeEach?

Reply:
You are right, it is not required again.